.travis.yml

```diff
@@ -119,8 +119,8 @@ after_success:
       echo "------- Build Documentation -------";
       python3 setup.py build_ext --inplace docs -j 2 -G "Unix Makefiles" || travis_terminate 1;
       cd $TRAVIS_BUILD_DIR/_skbuild/linux-x86_64-3.8/setuptools/sphinx;
-      rm -rf doctrees && touch html/.nojekyll;
```

Comment:
strange, locally I do not have .nojekyll and doctrees in the same folder

Reply:
I don't understand your comment?
You want to simply go into the build folder, delete all auxillary files, touch into that root the `.nojekyll` file and an `index.html` that forwards into `html/` and finally tell travis to force-push this root into github `gh-pages` branch